tests/financial/test_ny_stock_exchange.py

```
@@ -425,7 +425,7 @@ def test_special_holidays(self):
             "2025-01-09",  # National Day of Mourning for former President Jimmy Carter
         ]
 
-        def _make_special_holiday_list(begin, end, days=None, weekends=False):
+        def _make_special_holiday_list(begin, end, *, days=None, weekends=False):
```

Ruff (0.14.0)

Missing return type annotation for private function `_make_special_holiday_list`

(ANN202)